lib/board.rb

```diff
@@ -32,8 +37,45 @@ def free_positions
     flat_board.each_index.select { |index| empty?(flat_board[index]) }
   end
 
+  def content
```

Comment:
How is this method used elsewhere?
I am worried that it just exists to create a string which is then parsed again...